### ISSUE TITLE
Remove "Show Tab Bar"/"Show All Tabs" menu items on macOS

### ISSUE THREAD

**Issue by mjrodgers** [pulsar-edit/pulsar#647]
### Have you checked for existing feature requests?

- [X] Completed

### Summary

Running Pulsar on macOS, the `View` menu has two options at the top, `Show Tab Bar` and `Show All Tabs`. But these don't relate to the Pulsar tabs for files, they are for some kind of system-defined "app tabs" that aren't (to my knowledge?) implemented in Pulsar. 
![Screenshot 2023-07-18 at 11 25 38 AM](https://github.com/pulsar-edit/pulsar/assets/1795161/7ed0819d-f861-4da1-84a0-b59029d20726)

For example, if you `Show Tab Bar` it just gives an extra bar between the title bar and the actual tabs for open files.
![Screenshot 2023-07-18 at 11 26 56 AM](https://github.com/pulsar-edit/pulsar/assets/1795161/bbf82515-5582-4685-a894-b73a4f7cc5ba)

Presumably the intended behaviour (by Apple) would be that you could have different tabs here that play the role of different windows; so each of these tabs would house their own project. But this is not implemented. We can see that if we `Show All Tabs` the current project for the window is displayed, and a greyed-out option for adding a new "tab".
![Screenshot 2023-07-18 at 11 29 11 AM](https://github.com/pulsar-edit/pulsar/assets/1795161/78056743-4554-4dfb-bc55-7cf939d2df46)

It seems like these options should be removed from the View menu on macOS.

### What benefits does this feature provide?

Reduces menu clutter, also avoids having confusing menu items that don't provide any functionality.

### Any alternatives?

I guess an alternative would be to implement these features, and let these system "tabs" each hold their own Project. Then you could have your open Pulsar projects all in one application window instead of having a separate window for each (if you wanted to do this).

**Comment by confused-Techie** [comment]
Thanks a ton for reporting this one!

@mjrodgers So I took a quick look at this, admittedly I don't have a macOS machine to access, but I just want to confirm that you've launched Pulsar in Safe Mode?

I can't find any option being added to the menu bar from either [core](https://github.com/pulsar-edit/pulsar/blob/master/menus/darwin.cson#L130) or the [tabs package](https://github.com/pulsar-edit/pulsar/blob/master/packages/tabs/menus/tabs.cson) that adds this specific option.

Which makes me believe it may be caused by a community package. Although, admittedly there have been a few issues logged to Atom previously where users had trouble with this option disappearing, and updates to Atom seemed to always be the solution, without much if any explanation.

So this very well might be a built in feature that we can remove, I just can't seem to find where in the repo it actually exists. So if anyone has any insight that'd be awesome

**Comment by mjrodgers** [comment]
Yes, it is absolutely still there in safe mode.

 I think actually it appears on macOS by default unless it's explicitly disabled.

Here is some info about the issue:
https://github.com/electron/electron/pull/7415￼
Hide Show Tab Bar menu item on macOS Sierra by kevinsawicki · Pull Request #7415 · electron/electron
github.com

**Comment by savetheclocktower** [comment]
Yeah, this looks like [an Electron thing](https://stackoverflow.com/questions/73408994/electron-automatically-adds-hide-tab-bar-and-show-all-tabs-to-view-menu-on).

What's odd is that the PR @mjrodgers linked is supposed to have fixed this years ago, but we're seeing this on Electron 12.

**Comment by mjrodgers** [comment]
Yeah, definitely interesting. Not sure if it's a regression though (the issue is also on Atom 1.60.0 running Electron 9).

**Comment by savetheclocktower** [comment]
This may have worked itself out with the Electron 30 upgrade. I no longer see these menu items in my View menu.

@mjrodgers, can you confirm?

**Comment by mjrodgers** [comment]
Not resolved for me, they still appear and give weird behavior if I use them (Pulsar 1.131.1 and latest macOS Tahoe)
